system/jlib/jfile.cpp

```diff
@@ -102,6 +102,10 @@ static inline bool isPCFlushAllowed();
 
 static char ShareChar='$';
 
+enum expertFileFlagsEnum { NO_MASK, POS_MASK, NEG_MASK };
+static expertFileFlagsEnum hasExpertFileFlags = NO_MASK;
+static IFEflags expertFileFlagsMask;
```

Comment:
minor: even though should always be initialized before used, looks cleaner if also initialize to IFEnone here.